packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -188,8 +200,11 @@ contract Deploy is Deployer {
         deployOptimismMintableERC20FactoryProxy();
         deployL1ERC721BridgeProxy();
         deployDisputeGameFactoryProxy();
-
-        transferAddressManagerOwnership(); // to the ProxyAdmin
+        transferControl({
+            _controlled: "AddressManager",
+            _newController: "ProxyAdmin",
+            _transferSelector: Ownable.transferOwnership.selector
+        });
```

Comment:
Can we bring back the higher level functions like `transferAddressManagerOwnership` and then just wrap `transferControl`?

Reply:
Looks like this one comment was resolved without addressing

Reply:
oh sorry, I fixed it locally, but that change got lost in the shuffle. Going to wait for #7670 to merge, then I'll rebase, rebuild and push.

Reply:
Pushed the fix